importer/generate.js

```diff
@@ -92,6 +94,9 @@ function processFolder(srcPath, destPath) {
             } else {
               generateSelector(destPath, name)
             }
+            if (TARGET === 'react') {
+              generateReact(destPath, file.substring(0, file.length - 4), srcFile)
```

Comment:
Sorry one last request, instead of `file.length - 4`, could you follow the pattern in if branches above and use `file.lastIndexOf(SVG_EXTENSION)`?